test_scripts/Smoke/API/023_UnsubscribeButton_PositiveCase.lua

```diff
@@ -0,0 +1,113 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: UnsubscribeButton
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [UnsubscribeButton] SUCCESS: getting SUCCESS:UnsubscribeButton()
+--
+-- Description:
+-- Mobile application sends valid UnsubscribeButton request and gets UnsubscribeButton "SUCCESS" response from SDL
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests UnsubscribeButton with valid parameters
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if Buttons interface is available on HMI
+-- SDL checks if UnsubscribeButton is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL sends the Buttons notificaton to HMI
+-- SDL responds with (resultCode: SUCCESS, success:true) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Variables ]]
+local isMediaApplication = commonSmoke.GetAppMediaStatus()
```

Comment:
@GetmanetsIrina Propose to remove variability from the test.
Lets assume that the test will be for media application only, since this type of application is default one in config

Reply:
Removed.